Content.Client/_DV/AACTablet/UI/AACWindow.xaml.cs

```diff
@@ -0,0 +1,237 @@
+            .OrderBy(g => g.Key)
+            .ToDictionary(
+                g => g.Key,
+                g => g.GroupBy(p => p.Group)
+                    .OrderBy(gg => gg.Key)
+                    .ToDictionary(
+                        gg => gg.Key,
+                        gg => gg.OrderBy(p => Loc.GetString(p.Text)).ToList()
+                    )
+            );
+
+        CreateTabContainer(sortedTabs);
+    }
+
+    private void CreateTabContainer(Dictionary<string, Dictionary<string, List<QuickPhrasePrototype>>> sortedTabs)
+    {
+        foreach (var tab in sortedTabs)
+        {
+            var tabName = Loc.GetString(tab.Key);
+            var boxContainer = CreateBoxContainerForTab(tab.Value);
+            var scroll = new ScrollContainer();
+            scroll.HScrollEnabled = false;
+            scroll.AddChild(boxContainer);
+            WindowBody.AddChild(scroll);
+            WindowBody.SetTabTitle(WindowBody.ChildCount - 1, tabName);
+        }
+    }
+
+    private BoxContainer CreateBoxContainerForTab(Dictionary<string, List<QuickPhrasePrototype>> groups)
+    {
+        var boxContainer = new BoxContainer
+        {
+            HorizontalExpand = true,
+            Orientation = BoxContainer.LayoutOrientation.Vertical
+        };
+
+        foreach (var group in groups)
+        {
+            var header = CreateHeaderForGroup(group.Key);
+            var buttonContainer = CreateButtonContainerForGroup(group.Value);
+            boxContainer.AddChild(header);
+            boxContainer.AddChild(buttonContainer);
+        }
+
+        return boxContainer;
+    }
+
+    private static Label CreateHeaderForGroup(string groupName)
+    {
+        var header = new Label
+        {
+            HorizontalExpand = true,
+            Text = groupName,
+            Margin = new Thickness(10, 10, 10, 0),
+            StyleClasses = { "LabelBig" }
+        };
+
+        return header;
+    }
+
+    private GridContainer CreateButtonContainerForGroup(List<QuickPhrasePrototype> phrases)
+    {
+        var buttonContainer = CreateButtonContainer();
+        foreach (var phrase in phrases)
+        {
+            var text = Loc.GetString(phrase.Text);
+            var button = CreatePhraseButton(text, phrase.StyleClass);
+            button.OnPressed += _ => OnPhraseButtonPressed(new ProtoId<QuickPhrasePrototype>(phrase.ID));
+            buttonContainer.AddChild(button);
+        }
+        return buttonContainer;
+    }
+
+    private static GridContainer CreateButtonContainer()
+    {
+        var buttonContainer = new GridContainer
+        {
+            Margin = new Thickness(10),
+            Columns = ColumnCount
+        };
+
+        return buttonContainer;
+    }
+
+    private static Button CreatePhraseButton(string text, string styleClass)
+    {
+        var phraseButton = new Button
+        {
+            Access = AccessLevel.Public,
+            MaxSize = new Vector2(ButtonWidth, ButtonWidth),
```

Comment:
it might be worth playing around with this a little? the window is resizable but it looks pretty bad when scaled up since the buttons don't expand along with it. a `MinSize` might be better, i dunno